test/integration/cni/pod_traffic_across_az_test.go

```diff
@@ -142,23 +145,24 @@ var _ = Describe("[STATIC_CANARY] test pod networking", FlakeAttempts(retries),
 })
 
 // Function for AZ to Pod mapping and AZ to AZ ID mapping
-func GetAZMappings(nodes coreV1.NodeList) (map[string]coreV1.Pod, map[string]string) {
+func GetAZMappings(nodes coreV1.NodeList) (map[string]coreV1.Pod, map[string]string, error) {
 	// Map of AZ name to Pod from Daemonset running on nodes
 	azToPod := make(map[string]coreV1.Pod)
 	// Map of AZ name to AZ ID
 	azToazID := make(map[string]string)
+	nodePresenceInAZ := make(map[string]bool)
```

Comment:
It remains strange to me that Golang does not just have a set data structure in the standard library